doc/source/user/basics.creation.rst

```diff
@@ -114,33 +334,49 @@ convert are those formats supported by libraries like PIL (able to read and
 write many image formats such as jpg, png, etc).
 
 Common ASCII Formats
-------------------------
+--------------------
+
+Delimited files such as comma separated value (csv) and tab separated
+value (tsv) files are used for programs like Excel and LabView. Python
+functions can read and parse these files line-by-line. NumPy has two
+standard routines for importing a file with delimited data :func:`numpy.loadtxt`
+and :func:`numpy.genfromtxt`. These functions have more involved use cases in
+:doc:`how-to-io`. A simple example given a ``simple.csv``::
+
+ $ cat simple.csv
```

Comment:
```suggestion
:doc:`how-to-io`. A simple example given a ``simple.csv``:

.. code-block:: bash

 $ cat simple.csv
```
else this will be treated as python